go/ct/gen/state_test.go

```diff
@@ -434,3 +434,19 @@ func TestStateGenerator_ReturnDataShouldBeEmpty(t *testing.T) {
 		t.Errorf("unexpected length of generated return data, wanted %d, got %d", want, got)
 	}
 }
+
+// //////////////////////////////////////////////////////////
+// Block number hashes
+func TestStateGenerator_BlockNumberHashes(t *testing.T) {
+	newHashes := []vm.Hash{}
+	state := genRandomState(t)
+	for i := 0; i < 256; i++ {
+		if state.RecentBlockHashes[i] == (vm.Hash{}) {
+			t.Error("unexpected hash value, should not be zero")
+		}
+		if slices.Contains(newHashes, state.RecentBlockHashes[i]) {
```

Comment:
I think we don't need this test, even if this test pass, this does not prevent the random number generator from making the same sequence in runtime,

Reply:
?

Reply:
sorry. I forgot to reply. Yes, you are right, the generator could potentially generate a hash that is all zeros, and that would not be an error. hence the test was removed.